src/services/blocks/BlocksService.ts

```diff
@@ -701,14 +701,35 @@ export class BlocksService extends AbstractService {
 				} else if (argument instanceof GenericCall) {
 					newArgs[paramName] = this.parseGenericCall(argument, registry);
 				} else if (paramName === 'call' && argument?.toRawType() === 'Bytes') {
-					// multiSig.asMulti.args.call is an OpaqueCall (Vec<u8>) that we
+					// multiSig.asMulti.args.call is either an OpaqueCall (Vec<u8>),
+					// WrapperKeepOpaque<Call>, or WrapperOpaque<Call> that we
 					// serialize to a polkadot-js Call and parse so it is not a hex blob.
 					try {
 						const call = registry.createType('Call', argument.toHex());
 						newArgs[paramName] = this.parseGenericCall(call, registry);
 					} catch {
 						newArgs[paramName] = argument;
 					}
+				} else if (
+					paramName === 'call' &&
+					argument?.toRawType() === 'WrapperKeepOpaque<Call>'
+				) {
+					try {
+						const call = registry.createType('Call', argument.toHex());
+						newArgs[paramName] = this.parseGenericCall(call, registry);
+					} catch {
+						newArgs[paramName] = argument;
+					}
+				} else if (
+					paramName === 'call' &&
+					argument?.toRawType() === 'WrapperOpaque<Call>'
+				) {
+					try {
+						const call = registry.createType('Call', argument.toHex());
+						newArgs[paramName] = this.parseGenericCall(call, registry);
+					} catch {
+						newArgs[paramName] = argument;
+					}
```

Comment:
Ah ok, so basically these bits are the same as something like the following then I guess?:

```
} else if (argument && paramName === 'call' && ['Bytes', 'WrapperKeepOpaque<Call>', 'WrapperOpaque<Call>'].includes(argument.toRawType())) {
	// multiSig.asMulti.args.call is either an OpaqueCall (Vec<u8>),
	// WrapperKeepOpaque<Call>, or WrapperOpaque<Call> that we
	// serialize to a polkadot-js Call and parse so it is not a hex blob.
	try {
		const call = registry.createType('Call', argument.toHex());
		newArgs[paramName] = this.parseGenericCall(call, registry);
	} catch {
		newArgs[paramName] = argument;
	}
} else {
	newArgs[paramName] = argument;
}
```

Reply:
Yea that is way cleaner, ill change it to reflect that. Thanks!

Reply:
fixed